src/components/movieTitle/movieTitle.tsx

```diff
@@ -0,0 +1,27 @@
+import './movieTitle.css';
+
+interface MovieTitleProps {
+  imageUrl: string;
+  movieName: string;
+  releaseYear: number;
+  relevantGenres: Genre[];
+  onMovieSelect: () => void;
+}
+
+const MovieTitle = (props: MovieTitleProps) => {
+  return (
+    <div className="movie">
+      <img
+        className="movieImg"
+        src={props.imageUrl}
+        onClick={props.onMovieSelect}
+      />
+      <div className="movieInfo">
+        <h5 className="info">{props.movieName}</h5>
+        <h5 className="info withBorder">{props.releaseYear}</h5>
+        <h5 className="info">{props.relevantGenres.join(' & ')}</h5>
```

Comment:
Wrong formatting. If you are using the array methods you need to be sure that relevantGenres is an array. Otherwise if the part of data will be missing (undefined or null) the app will be broken

react-dom_client.js?v=b40794b0:19407 Uncaught TypeError: Cannot read properties of null (reading 'join')
    at MovieTitle (movieTitle.tsx?t=1709972780303:47:88)
    at renderWithHooks (react-dom_client.js?v=b40794b0:12169:26)
    at mountIndeterminateComponent (react-dom_client.js?v=b40794b0:14919:21)
    at beginWork (react-dom_client.js?v=b40794b0:15900:22)
    at beginWork$1 (react-dom_client.js?v=b40794b0:19747:22)
    at performUnitOfWork (react-dom_client.js?v=b40794b0:19192:20)
    at workLoopSync (react-dom_client.js?v=b40794b0:19131:13)
    at renderRootSync (react-dom_client.js?v=b40794b0:19110:15)
    at recoverFromConcurrentError (react-dom_client.js?v=b40794b0:18730:28)
    at performConcurrentWorkOnRoot (react-dom_client.js?v=b40794b0:18678:30)